buildout.cfg

```diff
@@ -1,8 +1,15 @@
 [buildout]
+extensions = mr.developer
 develop = .
 parts = test py
 
 find-links = http://download.zope.org/distribution/
+auto-checkout = *
+
+sources-dir = src-dev
+
+[sources]
+zc.relation = git https://github.com/zopefoundation/zc.relation.git branch=py3-compat
```

Comment:
Is this because of the cyclic interdependendency between zc.relation and zc.relationship that I think I remember being mentioned in one of the other PRs?

Reply:
Exactly, otherwise the test will fail:
```
Traceback (most recent call last):
...
    from zc.relationship import interfaces
  File ".../zc.relationship/src/zc/relationship/interfaces.py", line 21, in <module>
    import zc.relation.interfaces
   ...
TypeError: Class advice impossible in Python3.  Use the @implementer class decorator instead.
```